client/components/google-my-business/stats-nudge.js

```diff
@@ -3,52 +3,82 @@
-					/>
-				</div>
-				<div className="google-my-business__stats-nudge-info">
-					<h1 className="google-my-business__stats-nudge-title">
-						{ translate( 'Reach more customers with Google My Business' ) }
-					</h1>
-					<h2 className="google-my-business__stats-nudge-description">
-						{ translate(
-							'Show up when customers search for businesses like yours on Google Search and Maps.'
-						) }
-					</h2>
-					<div className="google-my-business__stats-nudge-button-row">
-						<Button primary>{ translate( 'Start Now' ) }</Button>
-						<Button>{ translate( "I've Already Listed" ) }</Button>
+class GoogleMyBusinessStatsNudge extends Component {
+	static propTypes = {
+		trackNudgeShown: PropTypes.func.isRequired,
+		trackNudgeDismissed: PropTypes.func.isRequired,
+		trackNudgeStartNowClicked: PropTypes.func.isRequired,
+		trackNudgeAlreadyListedClicked: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	componentWillMount() {
+		this.props.trackNudgeShown();
+	}
+
+	render() {
+		return (
+			<DismissibleCard
+				className="google-my-business__stats-nudge"
+				preferenceName="google-my-business-nudge"
+				temporary
+				onClick={ this.props.trackNudgeDismissed }
+			>
+				<SectionHeader
+					className="google-my-business__stats-nudge-header"
+					label={ this.props.translate( 'Recommendation from WordPress.com' ) }
+				/>
+				<div className="google-my-business__stats-nudge-body">
+					<div className="google-my-business__stats-nudge-image-wrapper">
+						<img
+							className="google-my-business__stats-nudge-image"
+							src="/calypso/images/google-my-business/phone-screenshot.png"
+							alt={ this.props.translate( 'Your business with Google My Business' ) }
+						/>
+					</div>
+					<div className="google-my-business__stats-nudge-info">
+						<h1 className="google-my-business__stats-nudge-title">
+							{ this.props.translate( 'Reach more customers with Google My Business' ) }
+						</h1>
+						<h2 className="google-my-business__stats-nudge-description">
+							{ this.props.translate(
+								'Show up when customers search for businesses like yours on Google Search and Maps.'
+							) }
+						</h2>
+						<div className="google-my-business__stats-nudge-button-row">
+							<Button primary onClick={ this.props.trackNudgeStartNowClicked }>
+								{ this.props.translate( 'Start Now' ) }
+							</Button>
+							<Button onClick={ this.props.trackNudgeAlreadyListedClicked }>
+								{ this.props.translate( "I've Already Listed" ) }
+							</Button>
+						</div>
 					</div>
 				</div>
-			</div>
-		</DismissibleCard>
-	);
-};
+			</DismissibleCard>
+		);
+	}
+}
 
-export default localize( GMBStatsNudge );
+export default connect( () => ( {} ), {
+	trackNudgeShown: () => recordTracksEvent( 'calypso_test_google_my_business_stats_nudge_shown' ),
+	trackNudgeDismissed: () =>
+		recordTracksEvent( 'calypso_test_google_my_business_stats_nudge_dismissed' ),
```

Comment:
Shouldn't this function and Track event have a `clicked` suffix like other other actions?